core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -231,9 +231,7 @@ export class ResultPanelComponent {
    * @param response
    */
   private displayResultTable(resultData: ReadonlyArray<object>) {
-     if (resultData.length < 1) {
```

Comment:
simply removing this line will cause new error, when resultData is empty , `this.currentDisplayColumns = Object.keys(resultData[0]).filter(x => x !== '_id');` this line of code, down below, will run into error `ERROR TypeError: Cannot convert undefined or null to object`. We still should handle the no-result case, but in a more gentle way (other than throwing a error)

Reply:
how is this conversation resolved? I did not see relevant changes.